pyhyp/utils.py

```diff
@@ -73,7 +99,7 @@ def simpleOCart(nearFile, dh, hExtra, nExtra, sym, mgcycle, outFile, userOptions
     hypOptions = {
         "patches": patches,
         "unattachedEdgesAreSymmetry": True,
-        "outerFaceBC": "farfield",
+        "outerFaceBC": "farfield",  # this is not important because it gets overwritten later anyways
```

Comment:
Would it be better to remove this option?

Reply:
I think if we need to play with pyhyp options for overset cases, it should be here..

Reply:
The BC will get overwritten because we need to add back the center block and re-connect. It is also easier to handle the BCs below because there is no ambiguity on what is being done on the center block. I will review this line and add a bit more explanation guiding people to use the keyword argument rather than this dictionary.